utils/packaging/CMakeLists.txt.in

```diff
@@ -37,17 +37,21 @@ message(STATUS "Using LVIM_VERSION: ${LVIM_VERSION}")
 include(GNUInstallDirs)
 
 # create and install lvim shim
-
 if(PACKAGE_FOR_WINDOWS OR WIN32)
+  set(LVIM_INPUT_BIN_NAME lvim.ps1)
   set(LVIM_BIN_NAME lvim.ps1)
+  set(BASE_DIR_VAR "\$(Resolve-Path \"\$PSScriptRoot\\..\\${CMAKE_INSTALL_DATAROOTDIR}\\lunarvim\")")
 else()
+  set(LVIM_INPUT_BIN_NAME lvim.template)
   set(LVIM_BIN_NAME lvim)
+  set(BASE_DIR_VAR "\$(readlink -f \"\$(dirname \"\$(realpath \"\$0\")\")/..\")/${CMAKE_INSTALL_DATAROOTDIR}/lunarvim")
```

Comment:
how about using [`PROJECT_SOURCE_DIR`](https://cmake.org/cmake/help/latest/variable/PROJECT_SOURCE_DIR.html)? unless you're referencing the destination, then it should just be using `CMAKE_INSTALL_xxx`.

if it's easier, you could make it "build" the scripts and then you can install them according to installation rules.

Reply:
it's for the destination, it needs to be relative for the appimage, the paths are random (e.g /tmp/.mount_lvim(1mk5W7d/usr/share/lunarvim)

Reply:
> it needs to be relative for the appimage

do you mean _inside_ the appimage rather than relative?  What I'm trying to say is that having `/../` somewhere in path doesn't necessarily make it relative. I don't see how `BASE_DIR_VAR` wouldn't look like this  `/some/path/../share/lunarvim`, _which might even get resolved when using `readlink`_.

Reply:
we can't use a fixed absolute path https://docs.appimage.org/reference/best-practices.html?highlight=relocatable#binaries-must-not-use-compiled-in-absolute-paths
the suggested way to deal with this is to make paths relative to the executable, so here the path is `PATH_TO_EXE/../${CMAKE_INSTALL_DATAROOTDIR}/lunarvim` (`/tmp/.mount_lvim(1mk5W7d/usr/bin/../share/lunarvim` in appimage, `/usr/bin/../share/lunarvim` in a normal installation)